src/main/java/org/sagebionetworks/web/client/view/ACTAccessApprovalsViewImpl.java

```diff
@@ -45,43 +54,42 @@
 	Div userSelectContainer;
 	@UiField
 	Div currentUserContainer;
-	
+	@UiField
+	Button exportButton;
+	@UiField
+	Anchor downloadLink;
 	private Presenter presenter;
 	private Header headerWidget;
-	
+	DateTimeUtils dateTimeUtils;
+	SynapseJavascriptClient jsClient;
 	Widget widget;
 	@Inject
 	public ACTAccessApprovalsViewImpl(ACTViewImplUiBinder binder,
-			Header headerWidget
+			Header headerWidget,
+			SynapseJavascriptClient jsClient,
+			DateTimeUtils dateTimeUtils
 			) {
 		widget = binder.createAndBindUi(this);
+		this.jsClient = jsClient;
+		this.dateTimeUtils = dateTimeUtils;
 		this.headerWidget = headerWidget;
 		headerWidget.configure();
 		
-		clearDateFilter.addClickHandler(new ClickHandler() {
-			@Override
-			public void onClick(ClickEvent event) {
-				presenter.onClearExpireBeforeFilter();
-			}
+		clearDateFilter.addClickHandler(event -> {
+			presenter.onClearExpireBeforeFilter();
 		});
-		clearUserFilter.addClickHandler(new ClickHandler() {
-			@Override
-			public void onClick(ClickEvent event) {
-				presenter.onClearUserFilter();
-			}
+		clearUserFilter.addClickHandler(event -> {
+			presenter.onClearUserFilter();
 		});
-		clearAccessRequirementFilter.addClickHandler(new ClickHandler() {
-			@Override
-			public void onClick(ClickEvent event) {
-				presenter.onClearAccessRequirementFilter();
-			}
+		clearAccessRequirementFilter.addClickHandler(event -> {
+			presenter.onClearAccessRequirementFilter();
 		});
 		
-		expiresBeforeDatePicker.addChangeDateHandler(new ChangeDateHandler() {
-			@Override
-			public void onChangeDate(ChangeDateEvent evt) {
-				presenter.onExpiresBeforeDateSelected(expiresBeforeDatePicker.getValue());
-			}
+		expiresBeforeDatePicker.addChangeDateHandler(event -> {
+			presenter.onExpiresBeforeDateSelected(expiresBeforeDatePicker.getValue());
+		});
+		exportButton.addClickHandler(event -> {
+			export(presenter.getExportData());
 		});
```

Comment:
Add my own click handler to downloadLink to release the blob reference (via Window.URL.revokeObjectURL(link.href))

Reply:
And flip back to Export button.